lib/rules/no-duplicate-selectors/__tests__/index.js

```diff
@@ -208,6 +208,15 @@ testRule(rule, {
 				},
 			],
 		},
+		{
+			code: 'textarea {}; input, textarea {}',
```

Comment:
```suggestion
			code: 'textarea {} input, textarea {}',
```

To make CSS valid :)